code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -608,7 +609,10 @@
 
 /datum/gas_reaction/pluox_formation/react(datum/gas_mixture/air, datum/holder)
 	var/list/cached_gases = air.gases
-	var/produced_amount = min(PLUOXIUM_FORMATION_MAX_RATE, cached_gases[/datum/gas/carbon_dioxide][MOLES], cached_gases[/datum/gas/oxygen][MOLES] * INVERSE(0.5), cached_gases[/datum/gas/tritium][MOLES] * INVERSE(0.01))
+	var/pressure = air.return_pressure()
+	var/minimal_mole_factor = min(cached_gases[/datum/gas/carbon_dioxide][MOLES], cached_gases[/datum/gas/oxygen][MOLES] * INVERSE(0.5), cached_gases[/datum/gas/tritium][MOLES] * INVERSE(0.01))
+	var/pressure_efficiency = 1 - NUM_E ** -(pressure / PLUOXIUM_FORMATION_PRESSURE_SCALE)
+	var/produced_amount = min(minimal_mole_factor * pressure_efficiency, cached_gases[/datum/gas/carbon_dioxide][MOLES], cached_gases[/datum/gas/oxygen][MOLES] * INVERSE(0.5), cached_gases[/datum/gas/tritium][MOLES] * INVERSE(0.01))
```

Comment:
same questions here

Reply:
Mostly same as above, but I'll give a short explanation for each of these.

Reaches 1 - inverse e efficiency when pressure reaches pressure scale. If it's 20% of pressure scale or below, you can safely assume it is near the pressure over pressure scale.